frontend/src/app/sessions/active-sessions/active-sessions.component.html

```diff
@@ -37,6 +37,13 @@ <h2 *ngIf="session.type === 'readonly'">Read-only session</h2>
         Tool: {{ session.version!.tool.name }} ({{ session.version!.name }})
       </span>
     </div>
+
+    <mat-card *ngIf="session.download_in_progress">
+      <mat-card-content>
+        <p>Your download is being prepared…</p>
+        <mat-progress-bar mode="indeterminate"></mat-progress-bar>
+      </mat-card-content>
+    </mat-card>
   </mat-card-content>
   <mat-card-actions class="flex-space-between">
```

Comment:
```suggestion
  <div class="flex-space-between">
```

`display: flex` is overwritten by `display: block` because of the higher CSS specificity, the buttons are not centered.